openproblems/tasks/denoising/methods/knn_smoothing.py

```diff
@@ -8,16 +8,17 @@
 
 # the decay and t parameter give knn smoothing behavior
 def _knn_smoothing(adata, solver):
-    import knn-smooth
-    
-    smoothX = knn-smooth.knn_smoothing(adata.X)
+    import knn - smooth
+
+    smoothX = knn - smooth.knn_smoothing(adata.X)
     adata.X = smoothX
     return adata
 
 
 def knn_smoothing(adata, test=False):
     return _knn_smoothing(adata)

```

Comment:
Since there's only one version of this method, there's no real reason to have the internal method -- you can just put the functionality directly in here.